### ISSUE TITLE
Issue during start: TypeError: arguments did not match any overloaded call

### ISSUE THREAD

**Issue by jdede** [rafaelpadilla/review_object_detection_metrics#130]
Installed everything in a venv and get the following error:

```
TypeError: arguments did not match any overloaded call:
  move(self, a0: QPoint): argument 1 has unexpected type 'float'
  move(self, ax: int, ay: int): argument 1 has unexpected type 'float'
```

To solve this, the float has to be converted to int.

**Comment by rafaelpadilla** [comment]
Hi @jdede 

Can you please provide the step-by-step installation and commands/steps to reproduce this error?

**Comment by jdede** [comment]
Sure: Used the virtual environment as available in PR #136 and performed the following steps:

```
python3 -m venv venv
. ./venv/bin/activate
pip install -r requirements.txt # (from PR #136)
python3 setup.py install
python3 run.py 
```
this results into the following error:

```
(venv) jd@tokio:~/src/github/review_object_detection_metrics$ python run.py 
Traceback (most recent call last):
  File "/home/jd/src/github/review_object_detection_metrics/run.py", line 13, in <module>
    ui = Main_Dialog()
         ^^^^^^^^^^^^^
  File "/home/jd/src/github/review_object_detection_metrics/src/ui/run_ui.py", line 37, in __init__
    self.center_screen()
  File "/home/jd/src/github/review_object_detection_metrics/src/ui/run_ui.py", line 44, in center_screen
    self.move(left, top)
TypeError: arguments did not match any overloaded call:
  move(self, a0: QPoint): argument 1 has unexpected type 'float'
  move(self, ax: int, ay: int): argument 1 has unexpected type 'float'
```

The PR #131 converts the floats to integer (as the coordinates are integer anyway) for two cases which fixes the above mentioned error.
